packages/manager/apps/container/src/container/nav-reshuffle/sidebar/navigation-tree/services/hostedPrivateCloud.ts

```diff
@@ -89,6 +89,19 @@ hostedPrivateCloudUniverse.children = [
         },
         features: ['veeam-cloud-connect'],
       },
+      {
+        id: 'hycu',
+        idAttr: 'hycu-link',
+        universe: hostedPrivateCloudUniverse.id,
+        translation: 'sidebar_hycu',
```

Comment:
**question:** Should not we add the translation content?

Reply:
Yes, it's on this PR : https://github.com/ovh/manager/pull/13256.

So it's true I not annotate this subtility